dom0/access_control.bats

```diff
@@ -35,6 +35,7 @@
 }
 
 @test "check SyncVM flask label" {
+    skip "SyncVM is not deployed by default."
```

Comment:
@dpsmith Not hard. It would be even easier to have a way to list VMs of a specific type with `xen-vm`. Until then, assuming the SyncVM is called `syncvm` (like the original test), probing for its type should work.